python/xt/test/test_hypothesis_la_generalized_eigensolver.py

```
@@ -0,0 +1,187 @@
+    throws Dune::Exception when LAPACKE is not linked (the types list would be empty). We
+    catch that here so the module-level MATRIX_CLASSES list stays empty and the test class
+    is skipped cleanly instead of failing at import time.
+    """
+    import dune.xt.la as la
+
+    solver_cls = getattr(la, cls.__name__ + "GeneralizedEigenSolver", None)
+    if solver_cls is None:
+        return False
+    try:
+        return "lapack" in solver_cls.types()
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)